apps/src/javalab/Javalab.js

```diff
@@ -8,6 +8,7 @@ import {lockContainedLevelAnswers} from '@cdo/apps/code-studio/levels/codeStudio
 import {TestResults} from '@cdo/apps/constants';
 import {EVENTS} from '@cdo/apps/metrics/AnalyticsConstants';
 import analyticsReporter from '@cdo/apps/metrics/AnalyticsReporter';
+import Neighborhood from '@cdo/apps/miniApps/neighborhood/Neighborhood';
```

Comment:
minor thing: I almost left a comment saying I thought we should consistently use absolute or relative imports for Neighborhood and NeighborhoodVisualizationColumn, but then I realized the (almost) circular references between the files in `javalab/neighborhood` and `miniapps/neighborhood`, ex. that

`apps/src/javalab/neighborhood/NeighborhoodVisualizationColumn.jsx`
uses
`apps/src/miniApps/neighborhood/NeighborhoodVisualization.tsx`

had me confused. Is it worth (eventually) adding a little README about how the mini-app is shared between Python and Java Lab or removing the `/neighborhood` folder in Java Lab now that there's just one file and its styles in there?

Reply:
The only reference to Java Lab in the mini app folder is the one translation file right? I left that so we didn't lose translations, but otherwise Java Lab wraps the miniApp.
I think it's ok to keep the neighborhood folder for now, it keeps neighborhood/theater at the same level.

Reply:
Also sorry @fisher-alice I hadn't seen your comments when I wrote mine. Yes, the plan is to wrap `NeighborhoodVisualization` in something in Python Lab. We can decide if we want to make that generic enough to be re-used by Java Lab as well (right now the Java Lab version has some Java Lab specifics like theme that I didn't want to mess with). I'll add a readme as a follow-up.

Reply:
> Yes, the plan is to wrap NeighborhoodVisualization in something in Python Lab. We can decide if we want to make that generic enough to be re-used by Java Lab as well (right now the Java Lab version has some Java Lab specifics like theme that I didn't want to mess with). I'll add a readme as a follow-up.

Yeah, it seems fine leaving `NeighborhoodVisualizationColumn` in `javalab` since it contains javalab-specific code. And if you do end up creating a wrapper generic enough for both labs, then remove.
I think the mini-app concept makes a lot of sense - maybe down the road if javalab is refactored, both neighborhood and theater could be couched in terms of mini-app?

Reply:
Yep they are considered mini-apps colloquially, but we hadn't used that term in code yet.